zips/zip-0317.rst

```diff
@@ -40,6 +52,8 @@ The goal of this ZIP is to change the conventional fees for transactions
 by making them dependent on the number of inputs and outputs in a transaction,
 and to get buy-in for this change from wallet developers, miners and Zcash users.
 
+Revision 1 of this ZIP additionally defines the fee mechanism associated with the Orchard Zcash Shielded Assets (OrchardZSA) protocol as described in ZIP 226 [#zip-0226]_ and ZIP 227 [#zip-0227]_.
```

Comment:
It doesn't strictly need to be a new Revision of ZIP 317 because it does not affect v5 transactions, and applying different rules for v6 transactions is non-disruptive to the existing specification because v6 transactions don't currently exist.